### ISSUE TITLE
Flaky test in SiteSidebar.test.tsx

### ISSUE THREAD

**Issue by mbertrand** [mitodl/ocw-studio#170]
### Steps to Reproduce

This test failed once on CI, so may fail again:

```
FAIL static/js/components/SiteSidebar.test.tsx
  ● Console

    console.error
      Warning: Encountered two children with the same key, `foobar`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
          in div (created by SidebarSection)
          in SidebarSection (created by SiteSidebar)
          in div (created by SiteSidebar)
          in SiteSidebar
          in Router
          in Provider
          in Provider (created by WrapperComponent)
          in WrapperComponent

      at printWarning (node_modules/react-dom/cjs/react-dom.development.js:88:30)
      at error (node_modules/react-dom/cjs/react-dom.development.js:60:5)
      at warnOnInvalidKey (node_modules/react-dom/cjs/react-dom.development.js:13801:11)
      at reconcileChildrenArray (node_modules/react-dom/cjs/react-dom.development.js:13832:21)
      at reconcileChildFibers (node_modules/react-dom/cjs/react-dom.development.js:14305:14)
      at reconcileChildren (node_modules/react-dom/cjs/react-dom.development.js:16762:28)
      at updateFragment (node_modules/react-dom/cjs/react-dom.development.js:16977:3)
      at beginWork (node_modules/react-dom/cjs/react-dom.development.js:18649:14)

  ● SiteSidebar › should pad all .config-sections excepting the last one

    expect(received).toEqual(expected) // deep equality

    - Expected  - 1
    + Received  + 0

      Array [
        "sidebar-section pb-4",
        "sidebar-section pb-4",
        "sidebar-section pb-4",
    -   "sidebar-section pb-4",
        "sidebar-section",
      ]

      75 |         .find("SidebarSection")
      76 |         .map(wrapper => wrapper.find("div").prop("className"))
    > 77 |     ).toEqual([
         |       ^
      78 |       "sidebar-section pb-4",
      79 |       "sidebar-section pb-4",
      80 |       "sidebar-section pb-4",

      at static/js/components/SiteSidebar.test.tsx:77:7
      at fulfilled (static/js/components/SiteSidebar.test.tsx:5:58)
 ```

**Comment by noisecapella** [comment]
I made a fix in a commit in #164

**Comment by pdpinch** [comment]
I think this can be closed.
